index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <title>mobile-project</title>
+</head>
+<body>
+  <header>
+  <nav class="flex-nav">
+    <div>
+    <img src="images/MyLogo.png" alt="logo">
+    </div>
+    <div>
+    <img src="images/Portoflio.png" alt="Portoflio" class="right">
+    <img src="images/About.png" alt="About" class="right">
+    <img src="images/Contact.png" alt="Contact" class="right">
+  </div>
+  </nav>
+</header>
+  <main class="flex-main">
+    <section class="flex-center">
+   <h1>I’m Alexandra<br>
+    Glad to see <br> you!</h1>
+    <p class="">I’m a software developer! I can help you build a product , feature or we
```

Comment:
- [ ] Please remove this empty `class` from the element, as it doesn't do anything, it affects the clarity of your code and takes up unnecessary space.

Reply:
These issues raised by the previous TSE still persist. Please make sure you made all the requested changes